### ISSUE TITLE
mat-datepicker default date status is not conveyed contextually

### ISSUE THREAD

**Issue by kesava885** [angular/components#16316]
#### Reproduction

Use StackBlitz to reproduce your issue: https://components-issue-gwv4xh.stackblitz.io

Steps to reproduce:
1.  Need a screen reader like JAWS to reproduce with issue.
2. Initialize mat-datepicker without any date
```html
<mat-form-field>
  <input matInput [matDatepicker]="picker" placeholder="Choose a date">
  <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
  <mat-datepicker #picker></mat-datepicker>
</mat-form-field>
```
3. Launch the datepicker in a browser with screen reader enabled.
 
 
#### Expected Behavior

 A simple update to the markup by including selected status to assist the screen reader in providing context of the selected date to the user. This could be done by including the word "selected" in the aria-label for that specific table cell.

#### Actual Behavior

When the mat-datepicker used with out any date value, angular selects the date of that day to present it to the user. This can be seen visually. But not being announced with the screen reader, for instance the sighted user can see what date is selected but the user using the screen reader does not. This made 508 test to fail. 

#### Environment

- Angular: 7
- CDK/Material: 7
- Browser(s): IE, Chrome
- Operating System: Windows

**Comment by jelbourn** [comment]
When no value is specified, focus will default to "today", but it is _not_ selected. The visual indicator here does not represent selection; it's only a marker that that date is today. Selection is indicated by `aria-selected` on the cell.

The most I think we should do here is add an `aria-describedby` to that cell with the word "Today", but this would require an extra i18n message.

Can you confirm that your issue is only for selection state?

**Comment by kesava885** [comment]
Yes, I can confirm that. Anything that helps screen reader to identify indication would be great.

**Comment by jelbourn** [comment]
If it's only for selection state, then it's currently working as intended (since that date is explicitly _not_ selected).
